### ISSUE TITLE
Allways didn't work.

### ISSUE THREAD

**Issue by zxl777** [sodanakin/Anonymous_Scripter_Tool#3]
Attempt 1.
Attempt 1 didn't work.
Attempt 2.
Attempt 2 didn't work.
Attempt 3.
Attempt 3 didn't work.
Attempt 4.
Attempt 4 didn't work.

**Comment by thatsweater** [comment]
Im having the same issue.

**Comment by Rawr2134** [comment]
i am also having the same issue =(

**Comment by sodanakin** [comment]
Hi this tool was originally made by Anonymous_Scripter on reddit. I just hosted it here so it wouldn't get lost (It was a pastebin originally) Sadly I don't have any Python experience and can't really add or solve issues =\

**Comment by phr34k0** [comment]
it's pretty useless as it is now honestly considering you need to verify those account to even use after 2days period.

**Comment by sodanakin** [comment]
@phr34k0  I agree. Oh well still leaving it up. Maybe someone knows how to fix it one day. I used this when it was working to get over 1k accounts.

**Comment by Sadye** [comment]
Verifying is included. @phr34k0

**Comment by phr34k0** [comment]
@Sadye found another https://github.com/MRokas/PogoPlayer
